moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -26,14 +30,31 @@ mapboxgl.workerClass =
 // returns geojson of features across all components
 const useProjectFeatures = (components) =>
   useMemo(() => {
+    if (components.length === 0)
```

Comment:
This fallback value patches a Mapbox GL warning about sources not having valid GeoJSON - an empty feature collection makes it happy.